typings/lerna__package-graph.d.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * skeleton only representing what we use.
+ */
```

Comment:
Can we pull any of these typings in from `DefinitelyTyped` rather than floating them ourselves?

Reply:
They don't currently exist there. What I've added here is pretty sparse coverage for the relevant packages. I was hoping to avoid taking on the load of publishing them there which I feel like would require a more comprehensive effort (also, never done it before...)